src/helpers/user.js

```diff
@@ -2,26 +2,32 @@ import ISO6391 from 'iso-639-1';
 import countries from 'svg-country-flags/countries.json';
 
 export function isMentor(user) {
-  return user.roles.includes('MENTOR');
+  try {
+    return user.roles.includes('MENTOR');
+  } catch {
+    return;
```

Comment:
I guess that the `catch` is catching when user is `undefined`, if so, maybe we can do the check instead of getting an exception?

Reply:
fixed, this is a perfect case for supporting the optional chaining operator